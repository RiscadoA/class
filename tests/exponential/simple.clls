proc main() {
    cut {
        println("!"); !a(b: close);
        println("close"); close b
        |a: ?wait|
        println("?"); ?a;
        println("call 1"); call a(y);
        println("wait 1"); wait y;
        println("call 2"); call a(z);
        println("wait 2"); wait z;
        ()
    }
};;
