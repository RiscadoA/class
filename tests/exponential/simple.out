!
?
call 1
close
wait 1
call 2
close
wait 2
